### DISCUSSION TITLE
Delete All Button in Trash Does Not Work

### DISCUSSION THREAD

**Discussion by Usr115** [immich-app/immich#11162] [Q&A]
### The bug

Using the 'Delete All' to delete large amounts of files will not select all images in trash, and will not delete them.  It will ask if you're sure you want to empty the trash, then giving a random number of images that were deleted once confirmed.  Refreshing the page, or leaving Trash and going back in, the images will be back, and not deleted.
The workaround so far is to select a few images to delete at a time.

### The OS that Immich Server is running on

Unraid 6.12.10

### Version of Immich Server

v1.108.0

### Version of Immich Mobile App

v.1.108.0 build.148

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
From Unraid CA Apps Page
```

### Your .env content

```Shell
N/A
From Unraid CA Apps Page
```

### Reproduction steps

```bash
1. Open WebUI for Immich
2. Send large number of files to Trash
3. Select, 'Trash'
4. Select, 'Delete All'
5. Confirm deletion
6. Refresh page
```

**Comment by alextran1502** [comment]
How many images did you have in the trash? It pushes the assets into a queue and deletes them to avoid hogging your IO stats

**Comment by Usr115** [comment]
I don't remember exactly, but it was a few hundred at the time.  It does show them queuing in the verbose logs, but left to sit for a few days had the same result.  I'm wondering if the size of the total files is just too much for it, since selecting a few at a time and deleting them works fine.

**Reply by jehadbaeth** [reply]
Hey, 
I encountered a similar problem after using [immich-duplicate-cleaner](https://github.com/BaptisteBuvron/immich-duplicate-cleaner) I ended up deleting 10K+ images and my setup went completely unresponsive. 

after checking logs from both immich_server and valkey I simply enabled overcommitting memory by running `sudo sysctl vm.overcommit_memory=1` and it fixed the issue for me.
